_posts/2020-03-07-Team-Retreat.md

```diff
@@ -0,0 +1,69 @@
+
+It is also important to have a retreat point of contact. Someone who will be in charge of planning and can answer questions regarding flights, accommodation, schedule, documentation, etc. This way, team members aren't lost seeking information.
+
+# The Agenda
+
+Once the retreat is planned and everything is booked, we need an agenda for the event. After all, we're having a once in a year opportunity to bring everyone to the same location.
+
+We like to open our retreats with a talk from our CEO on the company's vision and mission. This way, we all know where we're going.
+
+It is also important to plan your agenda carefully. A good way to get to great topics is to ask yourself "what challenges are we facing right now? And, of these, which ones impact us the most?". In a team retreat we have the opportunity to bring all heads together in the same room. It is a unique opportunity to foster discussion face-to-face, without the reservations we all have when interacting online. Therefore, make it matter.
+
+At our team retreat, we talked about company culture, discussed our mentorship programme, and gathered ideas to make our systems and processes better. 
+
+However, a good agenda must also include fun activities. A system we found effective was to intercalate presentations and discussions with fun team activities. This way, the day doesn't get boring and we can foster better discussions.
+
+# The Activities
+
+When planning activities in a team retreat, it is important to think of the value they bring. We want activities that will bring the team together while being entertaining, that will serve a clear purpose while being fun and enjoyable.
+
+Some activities we did include:
+
+- [SpaceTeam](http://playspaceteam.com)
+- [Escape Room in a Box](http://www.escaperoominabox.com/en-us)
+- Ask Me Anything
+- Two Truths and a Lie
+
+There is a great deal of flexibility with activities and lots of good games you can play with your team. Take advantage of it and choose games that will bring your team together.
+
+# Rules
+
+It is not all fun and games. It is important to understand the whole team will be together in one location for a company event. Even though we all want to have fun, it is important to keep it professional and make sure nobody is made uncomfortable. 
+
+In order to ensure things went smoothly, we came up with a Code of Conduct that was shared with all team members. This way, we established procedures for team members to follow if they felt uncomfortable for any reason.
+
+# Feedback
+
+After the retreat, it is good to get feedback. 
+
+We shared a feedback form with the whole team giving people the chance to talk about our choice of destination, the activities and the experience itself. What was amazing? What could we do better? This feedback will help us plan better retreats and grow closer as a company
```

Comment:
@ABizzinotto Not sure if you want to mention it, but I think it was positive to include so time off during the retreat. That way the team could connect on a personal level in an unstructured way. For example: I love that the team got together at the beach and shared songs that they liked/disliked :)

Reply:
@etagwerker I added a Time Off section. Could you review?